modal/cls.py

```diff
@@ -354,7 +354,7 @@ def _hydrate_metadata(self, metadata: Message):
                     self._method_functions[method_name] = _Function._new_hydrated(
                         self._class_service_function.object_id, self._client, method_handle_metadata
                     )
-        elif self._class_service_function:
+        elif self._class_service_function and self._class_service_function.object_id:
```

Comment:
I think you can use `.is_hydrated`.

Reply:
Can we add a comment of when we are expected to hit this branch?

Could there still be cases for 0.63+ where the `class_service_function` is defined, but not yet hydrated, when we hit this code path?

Reply:
> Could there still be cases for 0.63+ where the class_service_function is defined, but not yet hydrated, when we hit this code path?

Nope, that won't happen, which is why I decided not to include a comment, since we'll be deprecating 0.62 soon.